src/App.tsx

```diff
@@ -1,18 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const total = 42;
+  const [perPage, setPerPage] = useState(5);
+  const [curPage, setCurPage] = useState(1);
```

Comment:
```suggestion
  const [currentPage, setCurrentPage] = useState(1);
```
Don't forget to use more descriptive variables names